jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletChannel.java

```diff
@@ -465,7 +465,7 @@ public boolean handle()
 
                             Dispatcher dispatcher = new Dispatcher(getContextHandler(), uri, decodedPathInContext);
                             dispatcher.async(asyncContextEvent.getSuppliedRequest(), asyncContextEvent.getSuppliedResponse());
-                        });
+                        }, true);
```

Comment:
Can't we put the `try/finally` for calling the listeners inside the `dispatch() ->` block instead of inside `dispatch()` method?